staging/src/k8s.io/apiserver/pkg/storage/etcd3/errors.go

```diff
@@ -30,13 +31,41 @@ func interpretWatchError(err error) error {
 	return err
 }
 
-func interpretListError(err error, paging bool) error {
+const (
+	expired         string = "The resourceVersion for the provided list is too old."
+	continueExpired string = "The provided continue parameter is too old " +
+		"to display a consistent list result. You can start a new list without " +
+		"the continue parameter."
+	inconsistentContinue string = "The provided continue parameter is too old " +
+		"to display a consistent list result. You can start a new list without " +
+		"the continue paremeter, or use the continue token in this response to " +
+		"retrieve the remainder of the results. Continuing with the provided " +
+		"token results in an inconsistent list - objects that were created, " +
+		"modified, or deleted after the first chunk of results will be returned " +
+		"and the current time may show up in the list."
```

Comment:
@smarterclayton comments addressed. PTAL. Not sure what you meant with the "current time" here, the ListMeta doesn't have creationTimestamp.

Reply:
hrm, was supposed to be "between the time the first chunk was returned and now may show up in the list."

Reply:
Got it. PTAL. Thanks.